src/Mvc/Mvc.Formatters.Json/src/DependencyInjection/JsonMvcCoreBuilderExtensions.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Buffers;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.ObjectPool;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extensions methods for configuring MVC via an <see cref="IMvcCoreBuilder"/>.
+    /// </summary>
+    public static class JsonMvcCoreBuilderExtensions
+    {
+        /// <summary>
+        /// Configures Newtonsoft.Json specific features such as input and output formatters.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcCoreBuilder"/>.</param>
+        /// <returns>The <see cref="IMvcCoreBuilder"/>.</returns>
+        public static IMvcCoreBuilder AddJsonFormatters(this IMvcCoreBuilder builder)
```

Comment:
```suggestion
        public static IMvcCoreBuilder AddJsonFormatter(this IMvcCoreBuilder builder)
```

If you want to keep the method names consistent.